MCprep_addon/spawner/mcmodel.py

```diff
@@ -367,6 +367,12 @@ def add_model(
 			)
 
 			face.normal_update()
+			
+			# Give slight offset by normal for overlay geometry
+			if face_mat == "#overlay":
+				bmesh.ops.translate(bm, verts=face.verts,
+						    vec=0.05 * face.normal)
```

Comment:
Seems a little strong from my standpoint at 0.05, 

```suggestion
						    vec=0.02 * face.normal)
```

Not sure if that gets in the way of hte remove doubles distance, maybe drop that as well to half?